pkgs/development/compilers/openjdk/8.nix

```diff
@@ -215,6 +215,10 @@ let
       maintainers = with maintainers; [ edwtjo ];
       platforms = [ "i686-linux" "x86_64-linux" "aarch64-linux" ];
       mainProgram = "java";
+      # Broken for musl at 2024-01-17. Tracking issue:
+      # https://github.com/NixOS/nixpkgs/issues/281618
+      # error: ‘isnanf’ was not declared in this scope
+      broken = stdenv.hostPlatform.isMusl;
```

Comment:
Still broken?

Reply:
I confirm it is still broken. There are patches at Alpine but this is a rabbithole. I tried it a couple of times. For now, i think it's better to mark it broken.

Reply:
OK then.

We can do a spintable later.